game-frontend/src/components/unitstatus/index.css

```diff
@@ -0,0 +1,17 @@
+.circular-char {
```

Comment:
We don't use separate css files in the project. Check `CodeBlock.tsx` for example.